### PR TITLE
New example app - Search for a TV show

### PR DESCRIPTION
**Pull request by SidKH**
This PR adds a new app to the examples.
The app uses a public API from https://www.tvmaze.com/ to search for a TV show
 

https://twitter.com/kentcdodds/status/1469315865174708225

https://user-images.githubusercontent.com/7762253/145627666-a5432c16-c8ef-4162-9723-5a5115f1876d.mp4

### CODE REVIEW

File: examples/search-tv-show/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+```sh
+npm start
+```
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/basic)
```
Comment (kentcdodds) [inline examples/search-tv-show/README.md:36]: Could you update this to:

```suggestion
[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/search-tv-show)
```

File: examples/search-tv-show/app/utils.server.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Sometimes some modules don't work in the browser, Remix will generally be
```
Comment (kentcdodds) [inline examples/search-tv-show/app/utils.server.tsx:1]: Can you delete this file? We don't need it.

File: examples/search-tv-show/app/routes/index.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+};
+
+export default function Index() {
+  const data = useLoaderData();
+  const transition = useTransition();
+  
+  return (
+    <div className="container">
+      <Form method="get" action="/" className="search-form">
+        <input defaultValue={data.searchTerm} placeholder="Search a TV show..." autoComplete="off" name="search" type="text" />
```
Comment (kentcdodds) [inline examples/search-tv-show/app/routes/index.tsx:55]: I think this should be a `type="search"`

```suggestion
        <input defaultValue={data.searchTerm} placeholder="Search a TV show..." autoComplete="off" name="search" type="search" />
```

File: examples/search-tv-show/app/routes/index.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  
+};
+
+export default function Index() {
+  const data = useLoaderData();
+  const transition = useTransition();
+  
+  return (
+    <div className="container">
+      <Form method="get" action="/" className="search-form">
```
Comment (kentcdodds) [inline examples/search-tv-show/app/routes/index.tsx:54]: The `action` prop shouldn't be needed:

```suggestion
      <Form method="get" className="search-form">
```

**Review by kentcdodds — CHANGES_REQUESTED** [review]
Sweet! Looks mostly great. Just a few things. Thanks!

File: examples/search-tv-show/app/utils.server.tsx (hunk 2 above)
Comment (SidKH) [reply examples/search-tv-show/app/utils.server.tsx]: All done

File: examples/search-tv-show/app/root.tsx [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+        position: "absolute",
+        width: "1px",
+        whiteSpace: "nowrap",
+        wordWrap: "normal"
+      }}
+    >
+      {innerHtml}
+    </div>
+  );
+});
```
Comment (kentcdodds) [inline examples/search-tv-show/app/root.tsx:189]: This is something we had in our template for a while but we removed it. We're not ready to commit to this component in our templates right now. Could you remove that?
```suggestion
```

**Review by kentcdodds — APPROVED** [review]
Super! Just one last thing. I'll go ahead and commit/merge 👍

File: examples/search-tv-show/app/root.tsx [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+      <head>
+        <meta charSet="utf-8" />
+        <meta name="viewport" content="width=device-width,initial-scale=1" />
+        {title ? <title>{title}</title> : null}
+        <Meta />
+        <Links />
+      </head>
+      <body>
+        {children}
+        <RouteChangeAnnouncement />
```
Comment (kentcdodds) [inline examples/search-tv-show/app/root.tsx:64]:
```suggestion
```

**Review by kentcdodds — APPROVED** [review]
Thanks 👏

**Pull request merged**